internal/venice-common/src/test/java/com/linkedin/venice/utils/UtilsTest.java

```diff
@@ -252,6 +253,29 @@ public void testResolveKafkaUrlForSepTopic() {
     assertEquals(Utils.resolveKafkaUrlForSepTopic(originalKafkaUrl), originalKafkaUrl);
   }
 
+  @Test
+  void testGetRealTimeTopicNames() {
+    Store mockStore = mock(Store.class);
+    List<Version> mockVersions = new ArrayList<>();
+    mockVersions.add(mock(Version.class));
+    mockVersions.add(mock(Version.class));
+    mockVersions.add(mock(Version.class));
+    HybridStoreConfig mockHybridConfig = mock(HybridStoreConfig.class);
+
+    when(mockStore.getName()).thenReturn("TestStore");
```

Comment:
Let's also change the store name here to "StoreName" to be consistent with RT name.